lib/elixir/lib/kernel.ex

```diff
@@ -3711,6 +3711,17 @@ defmodule Kernel do
       target = Keyword.get(opts, :to) ||
         raise ArgumentError, "expected to: to be given as argument"
 
+      if is_list(funs) do
+        IO.write :stderr, "warning: passing a list to Kernel.defdelegate/2 is deprecated, " <>
```

Comment:
Actually both form were used, so I picked the elixir syntax.
https://github.com/elixir-lang/elixir/search?p=2&q=deprecated&type=Code&utf8=%E2%9C%93

I'll change it to `:elixir_errors.warn` if it is what should be used for deprecations.

I think a `deprecate` helper in Elixir would be useful, as it is probably something that will be done regularly.
What do you think?